rotkehlchen/db/eth2.py

```diff
@@ -543,16 +557,22 @@ def process_accumulating_validators_balances_and_pnl(
                 ):  # only keep related withdrawal requests
                     withdrawal_request_events[v_index].append(request_event)
 
+        balances_over_time.update(cached_balances_over_time)
+        withdrawals_pnl.update(cached_withdrawals_pnl)
+        exits_pnl.update(cached_exits_pnl)
+
         from_ts_ms, to_ts_ms = ts_sec_to_ms(from_ts), ts_sec_to_ms(to_ts)
         validator_balances: dict[int, FVal] = defaultdict(lambda: ZERO)
+        # track skimming withdrawals per timestamp to properly cache and filter by time range
```

Comment:
This still does not make sense. What you want to say is probably:
```suggestion
        # This mapping is for skimming withdrawals per timestamp to properly cache and filter by time range
```